pkg/reconciler/v1alpha1/revision/config/observability.go

```diff
@@ -84,5 +88,9 @@ func NewObservabilityFromConfigMap(configMap *corev1.ConfigMap) (*Observability,
 		oc.RequestLogTemplate = rlt
 	}
 
+	if mb, ok := configMap.Data["metrics.request-metrics-backend-destination"]; ok {
```

Comment:
do we need to provide default value here?

Reply:
No. Empty means not enabling request metrics from queue-proxy.